garage/tf/algos/batch_polopt.py

```diff
@@ -22,25 +46,6 @@ def __init__(self,
                  positive_adv=False,
                  fixed_horizon=False,
                  **kwargs):
```

Comment:
please don't swallow kwargs

Reply:
I didn't get your point. I didn't make any concrete change here.

Reply:
Now many algorithms are using `kwargs`. I worked on it for a while. I think we may need another issue/PR to update all of them.

Reply:
i think this is fair. please file an issue for auditing this throughout the codebase (you don't need to assign yourself).

however, please do fix it for the classes you touch in this PR. we often fix small issues like this as we go using the rule "if you touch it, you have to fix it."

Reply:
Updated for all on policy algorithms.